### PR TITLE
update test

### PR DESCRIPTION
**Pull request by dsikka**
SUMMARY:
"please provide a brief summary"

TEST PLAN:
"please outline how the changes were tested"

### CODE REVIEW

**Review by kylesayrs — COMMENTED** [review]
Seems to be another issue at play
```
ValueError: Trying to set a tensor of shape torch.Size([64, 1]) in "weight_scale" (which has shape torch.Size([64, 0])), this looks incorrect.
```

May want to run in a local environment to make sure that all targeted weights are divisible by the specified group size

**Comment by dsikka** [comment]
> Seems to be another issue at play
> 
> ```
> ValueError: Trying to set a tensor of shape torch.Size([64, 1]) in "weight_scale" (which has shape torch.Size([64, 0])), this looks incorrect.
> ```

Yup I was trying to figure out why this is showing up now. This PR fixes this. While doing that: https://github.com/neuralmagic/compressed-tensors/pull/178  I decided to also enhance test coverage, hence this PR.

File: tests/llmcompressor/transformers/gptq/test_oneshot.py [hunk 1]
```diff
@@ -36,6 +36,16 @@
     },
 )
 
+recipe_modifier_full_group = GPTQModifier(
+    ignore=["lm_head"],
+    sequential_update=False,
```
Comment (kylesayrs) [inline tests/llmcompressor/transformers/gptq/test_oneshot.py:41]:
```suggestion
```

File: tests/llmcompressor/transformers/gptq/test_oneshot.py [hunk 2]
```diff
@@ -76,7 +87,8 @@ def test_oneshot_application(self):
             num_calibration_samples=9,
         )
 
-        model_loaded = SparseAutoModelForCausalLM.from_pretrained(self.output)
+
```
Comment (kylesayrs) [inline tests/llmcompressor/transformers/gptq/test_oneshot.py:90]:
```suggestion
```

File: tests/llmcompressor/transformers/gptq/test_oneshot.py [hunk 3]
```diff
@@ -106,6 +118,7 @@ def test_oneshot_application(self):
         # Check lm-head is not quantized
         not_targetted = model_loaded.lm_head
         assert not hasattr(not_targetted, "quantization_scheme")
+       
 
     def tearDown(self):
-        shutil.rmtree(self.output)
+         shutil.rmtree(self.output)
```
Comment (kylesayrs) [inline tests/llmcompressor/transformers/gptq/test_oneshot.py:124]:
```suggestion
        shutil.rmtree(self.output)
```

**Review by kylesayrs — APPROVED** [review]

**Review by mgoin — APPROVED** [review]

**Pull request merged**